soccer/MainWindow.cpp

```diff
@@ -31,6 +31,8 @@ using namespace Eigen;
 QString LiveStyle("border:2px solid transparent");
 QString NonLiveStyle("border:2px solid red");
 
+std::vector<QString> hiddenLayers {"MotionControl", "Planning0", "Planning1", "Planning2", "Planning3", "Planning4", "Planning5", "time", "velocity"};
```

Comment:
This doesn't reflect the current visibility of layers in the UI, only the default/initial visibility.  Maybe rename it initiallyHiddenLayers?  Also, should probably be an ivar, not a global.